app/models/concerns/hostext/search.rb

```diff
@@ -68,6 +68,7 @@ module Search
 
       scoped_search :relation => :primary_interface, :on => :ip, :complete_value => true
       scoped_search :relation => :interfaces, :on => :ip, :complete_value => true, :rename => :has_ip, :only_explicit => true
+      scoped_search :relation => :interfaces, :on => :ip6, :complete_value => true
```

Comment:
I would go a step further, and create two search conditions: `has_ip6`, similar to:
```
      scoped_search :relation => :interfaces, :on => :ip, :complete_value => true, :rename => :has_ip, :only_explicit => true
```
and `ip6` similar to:
```
      scoped_search :relation => :primary_interface, :on => :ip, :complete_value => true
```

This will get all our bases covered